rest-api/dao/dv_order_dao.py

```diff
@@ -24,6 +24,11 @@ class DvOrderDao(UpdatableDao):
   def __init__(self):
     self.code_dao = CodeDao()
     super(DvOrderDao, self).__init__(BiobankDVOrder)
+    self.biobank_address = {'city': "Rochester", 'state': "MN",
```

Comment:
This looks like something used in testing that was left in.

Reply:
I believe that is the real address of the Biobank where samples are shipped to.

Reply:
correct, it was intended to be left there.